src/index.html.ejs

```diff
@@ -38,16 +38,14 @@
   <div class="map" id="map"></div>
 
   <!-- First draw an empty map -->
-  <!-- FIXME Doesn't support asset hashing -->
-  <script src="/initialize-map.js"></script>
+  <script src="<%= htmlWebpackPlugin.files.chunks['initialize-map'].entry %>"></script>
 
   <!-- Then grab data dependencies -->
   <script src="https://s3.amazonaws.com/thirdbear-backups/aclu/public/action_events.js"></script>
-  <script src="./data/us_postal_codes.js"></script>
+  <script src="/us_postal_codes.js"></script>
```

Comment:
This file is huge and is unlikely to change -- skipping automatic hashing for it because that way slows down the build.